Source/AliveLib/Slurg.cpp

```diff
@@ -0,0 +1,29 @@
+#include "stdafx.h"
+#include "Slurg.hpp"
+
+WORD Slurg::sSlurg_Step_Watch_Points_Idx_5C1C08;
+char Slurg::sSlurg_Step_Watch_Points_Count_5BD4DC[2];
```

Comment:
Need to use ALIVE_VAR/ALIVE_ARY on these - might need to make them globals in the CPP for that to work